xcp_d/interfaces/regression.py

```diff
@@ -54,38 +56,60 @@ def _run_interface(self, runtime):
         # Get the confound matrix
         confound = pd.read_table(self.inputs.confounds)
 
-        confound = confound.to_numpy().T  # Transpose confounds matrix to line up with bold matrix
+        # Any columns starting with "signal__" are assumed to be signal regressors
```

Comment:
Need to document this feature in the custom confounds section.

Reply:
Done in [d1f9306](https://github.com/PennLINC/xcp_d/pull/697/commits/d1f9306ca14bf86fc10fac0dcbbcf4235adfb24a).